src/Octokit.Webhooks/WebhookEventProcessor.cs

```diff
@@ -65,1324 +66,1029 @@ public virtual void ProcessWebhook(IDictionary<string, StringValues> headers, st
                 case WebhookEventType.SecretScanningAlert:
-                    this.ProcessSecretScanningAlertWebhook(headers, body);
-                    break;
+                    return this.ProcessSecretScanningAlertWebhook(headers, body);
                 case WebhookEventType.SecurityAdvisory:
-                    this.ProcessSecurityAdvisoryWebhook(headers, body);
-                    break;
+                    return this.ProcessSecurityAdvisoryWebhook(headers, body);
                 case WebhookEventType.Sponsorship:
-                    this.ProcessSponsorshipWebhook(headers, body);
-                    break;
+                    return this.ProcessSponsorshipWebhook(headers, body);
                 case WebhookEventType.Star:
-                    this.ProcessStarWebhook(headers, body);
-                    break;
+                    return this.ProcessStarWebhook(headers, body);
                 case WebhookEventType.Status:
-                    this.ProcessStatusWebhook(headers, JsonSerializer.Deserialize<StatusEvent>(body)!);
-                    break;
+                    return this.ProcessStatusWebhook(headers, JsonSerializer.Deserialize<StatusEvent>(body)!);
                 case WebhookEventType.Team:
-                    this.ProcessTeamWebhook(headers, body);
-                    break;
+                    return this.ProcessTeamWebhook(headers, body);
                 case WebhookEventType.TeamAdd:
-                    this.ProcessTeamAddWebhook(headers, JsonSerializer.Deserialize<TeamAddEvent>(body)!);
-                    break;
+                    return this.ProcessTeamAddWebhook(headers, JsonSerializer.Deserialize<TeamAddEvent>(body)!);
                 case WebhookEventType.Watch:
-                    this.ProcessWatchWebhook(headers, body);
-                    break;
+                    return this.ProcessWatchWebhook(headers, body);
                 case WebhookEventType.WorkflowDispatch:
-                    this.ProcessWorkflowDispatchWebhook(headers, JsonSerializer.Deserialize<WorkflowDispatchEvent>(body)!);
-                    break;
+                    return this.ProcessWorkflowDispatchWebhook(headers, JsonSerializer.Deserialize<WorkflowDispatchEvent>(body)!);
                 case WebhookEventType.WorkflowJob:
-                    this.ProcessWorkflowJobWebhook(headers, body);
-                    break;
+                    return this.ProcessWorkflowJobWebhook(headers, body);
                 case WebhookEventType.WorkflowRun:
-                    this.ProcessWorkflowRunWebhook(headers, body);
-                    break;
+                    return this.ProcessWorkflowRunWebhook(headers, body);
             }
+
+            return Task.CompletedTask;
         }
 
-        private void ProcessBranchProtectionRuleWebhook(WebhookHeaders headers, string body)
+        private Task ProcessBranchProtectionRuleWebhook(WebhookHeaders headers, string body)
```

Comment:
Can you continue with the `Async` suffix all the way down?

```suggestion
        private Task ProcessBranchProtectionRuleWebhookAsync(WebhookHeaders headers, string body)
```

Reply:
Async suffix has been added to all the methods